test/service/test_manifest.py

```diff
@@ -1515,7 +1515,8 @@ def test_fetch_manifest(self, get_cached_manifest):
                     'CommandLine': {
                         'cmd.exe': None,
                         'bash': f"curl '{manifest_url}' | curl --config -"
-                    } if format_ == ManifestFormat.curl else {}
+                    } if format_ == ManifestFormat.curl else
```

Comment:
This is still not in the spirit of wrap all or none. It's oddly asymmetrical.

```suggestion
                    }
                    if format_ == ManifestFormat.curl else
```